gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/aggregation/functions/SetAggregateProperties.java

```diff
@@ -51,6 +56,7 @@ public SetAggregateProperties(final Set<AggregateFunction> aggregateFunctions) {
     }
 
     defaultValues = new HashMap<>();
+    this.aggregateFunctions = aggregateFunctions;
```

Comment:
Does not depend on your PR, but please put the null check of the first loop in the second loop and place your assignment below.